ingest-app/src/cmr/ingest/services/granule_bulk_update_service.clj

```diff
@@ -64,14 +109,12 @@
                    json-body
                    instructions)
                   (catch Exception e
-                    (error "validate-and-save-bulk-granule-update caught exception:" e)
-                    (let [message (.getMessage e)
-                          user-facing-message (if (string/includes? message "GBUT_PN_I")
-                                                "Granule bulk update task name needs to be unique within the provider."
-                                                message)]
+                    (error "An exception occurred saving a bulk granule update request:" e)
+                    (let [message (.getMessage e)]
                       (errors/throw-service-errors
-                       :invalid-data
-                       [(str "error creating granule bulk update task: " user-facing-message)]))))]
+                       :internal-error
+                       [(str "There was a problem saving a bulk granule update request."
+                             "Please try again, if the problem persists please contact cmr@nasa.gov.")]))))]
```

Comment:
I don't think `cmr@nasa.gov` is a valid email. I think it should be `cmr-support@earthdata.nasa.gov`. Please ask John to confirm.